src/js/views/patients/patient/sidebar/sidebar_views.js

```diff
@@ -61,21 +61,35 @@ const SidebarView = View.extend({
   },
   onClickMenu() {
     const canEdit = this.model.canEdit();
+    const patientStatus = this.model.getStatus();
+    const canManagePatients = this.getOption('canManagePatients');
 
     const menuOptions = new Backbone.Collection([
       {
+        text: canEdit ? i18n.menuOptions.edit : i18n.menuOptions.view,
         onSelect: bind(this.triggerMethod, this, canEdit ? 'click:patientEdit' : 'click:patientView'),
       },
     ]);
 
+    if (canEdit && canManagePatients) {
```

Comment:
is this really `canEdit`?  Can only manually added patients status be adjusted?

I wonder if really this check should be `this.model.getWorkspacePatient().canEdit()`

Reply:
Hmm right, probably don't need to check if it's a manually added patient.

In https://github.com/RoundingWell/care-ops-frontend/pull/1227/commits/11b4960b624c3b100536e0d2f743af89e3773186, I created a `workspacePatient.canEdit()` function and use that here instead. Where I check `isNew()` and the `currentUser.can('patient:manage')` permission.

Reply:
I guess I'm not sure if we need the isNew check.  I don't think this sidebar can be shown for a new patient, so we'll never be able to hit that coverage

Reply:
Okay, I removed the `isNew()` check from `workspacePatient.canEdit()`. Which now only checks that the current user has the `patient:manage` permission.